src/components/Feed.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState, useEffect} from "react";
+import Post from "../components/Post";
+import { Grid } from "@mui/material";
+
+function Feed(props) {
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:5000/posts')
```

Comment:
Why are posts being fetched on localhost? Shouldn't this be hooked up to heroku?